std/uni.d

```diff
@@ -622,7 +622,7 @@ import std.range.primitives; // back, ElementEncodingType, ElementType, empty,
     // save
 import std.traits; // isConvertibleToString, isIntegral, isSomeChar,
     // isSomeString, Unqual
-
+import std.exception; // enforce
```

Comment:
Yes, but then we discovered that non-selective imports are leaking:

https://github.com/dlang/phobos/pull/5584#issuecomment-314910297

tl;dr: if you add `std.exception` here, people can import all symbols from `std.exception` from `std.uni`